server/integration-tests/data-survives-restart.sh

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env bash
 
-if [ ! -f "stack.yaml" ]; then
-    echo "Run test in server package directory"
-    exit 1
-fi
+cd "$(dirname "$0")"
+cd ..
```

Comment:
Thanks I hate it (but can't think of why this is objectively bad)

Reply:
```shell
cd "$(dirname "$(dirname "$0")")"
```
doesn't work if the script is invoked relatively from the same directory or through a couple other names. This works, and I've done it in the Hoff repo, but it's a bit uglier so just doing the two `cd`s is probably better:

```bash
cd "$(dirname "$(dirname "$(realname "$0")")")"
```